components/bpmn/org.wso2.carbon.bpmn.analytics.publisher/src/main/java/org/wso2/carbon/bpmn/analytics/publisher/AnalyticsPublisher.java

```diff
@@ -0,0 +1,273 @@
+/*
```

Comment:
Need copyright date in the license text. Check the ide configuration and add the correct license text